.github/workflows/acceptance-tests.yml

```diff
@@ -61,8 +67,8 @@ jobs:
           role-duration-seconds: 21600 # 6h
       - run: |
           echo "Testing with Go ${{ needs.get-go-version.outputs.go-version }}"
-          PACKER_ACC=1 go test \
```

Comment:
I believe PACKER_ACC=1 is required here, otherwise acceptance tests will not run

Reply:
Since it's set from the environment above you can remove it from the command-line here, I think this is OK

Reply:
Oh derp, my bad.